src/Equinor.ProCoSys.IPO.Query/GetInvitationById/ParticipantDto.cs

```diff
@@ -1,20 +1,30 @@
-﻿using System;
+﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
 
 namespace Equinor.ProCoSys.IPO.Query.GetInvitationById
 {
     public class ParticipantDto
     {
-        public ParticipantDto(Guid? oid, string email, string name, MeetingResponse response)
+        public ParticipantDto(
+            Organization organization,
+            int sortKey,
+            ExternalEmailDto externalEmail,
+            PersonDto person,
+            FunctionalRoleDto functionalRole,
+            string rowVersion)
         {
-            Oid = oid;
-            Email = email;
-            Name = name;
-            Response = response;
+            Organization = organization;
+            SortKey = sortKey;
+            ExternalEmail = externalEmail;
+            Person = person;
+            FunctionalRole = functionalRole;
+            RowVersion = rowVersion;
         }
 
-        public Guid? Oid { get; }
-        public MeetingResponse Response { get; }
-        public string Email { get; }
-        public string Name { get; }
+        public Organization Organization { get; }
+        public int SortKey { get; }
+        public ExternalEmailDto ExternalEmail { get; }
+        public PersonDto Person { get; }
+        public FunctionalRoleDto FunctionalRole { get; }
+        public string RowVersion { get; }
```

Comment:
I think rowVersion needs to be on ExternalEmailDto, FunctionalRoleDto, and PersonDto instead of on ParticipantDto